### PR TITLE
DevKit updates for master branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: tests/autoload.php.dist_DELETE [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (OskarStark) [inline tests/autoload.php.dist_DELETE]: Hmm this should be deleted

File: phpunit.xml.dist [hunk 2]
```diff
@@ -9,11 +9,8 @@ It's auto-generated by sonata-project/dev-kit package.
 <phpunit backupGlobals="false"
          backupStaticAttributes="false"
          colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
          failOnWarning="true"
-         processIsolation="false"
```
Comment (OskarStark) [inline phpunit.xml.dist:16]: Ping @franmomu

**Comment by franmomu** [comment]
This one is fine, is the same error as in the last commit of `1.x` branch: https://github.com/sonata-project/SonataAdminSearchBundle/commit/7d2aef871fe0196b86eb319ef8845b52df1e291b

**Review by franmomu — DISMISSED** [review]

**Comment by OskarStark** [comment]
Yes I need to double check that 😬

**Comment by OskarStark** [comment]
@franmomu did the PHP 8 Tests pass before this PR?

**Pull request closed without merging**